docs/source/examples/checks/integrity/data_duplicates.ipynb

```diff
@@ -5,19 +5,28 @@
    "id": "06bdcdfc",
```

Comment:
<p>Maybe add - below the output - a sentence like </p><p>"As it can be seen, the condition found we have data duplicates in our dataset!"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>